packages/ffe-core/less/theme.less

```diff
@@ -174,6 +174,217 @@
 
     @media (prefers-color-scheme: dark) {
         .regard-color-scheme-preference {
+            /* SEMANTIC VARIABLES */
+            --ffe-color-background-default: var(--ffe-color-graa-1000);
```

Comment:
Disse css-variablene ser ikke ut til å finnes, fargene er kun definert i Less. Så for at dette skal funke må alle less-variablene oversettes til css i theme.less slik det var gjort tidligere. :)